### PR TITLE
Feature/study page design

### PR DESCRIPTION
**Pull request by mpgxvii**

### CODE REVIEW

File: src/app/studies/containers/studies-page.html [hunk 1]
```diff
@@ -1,35 +1,46 @@
...
-    <img class="logo" src="assets/radar-logo.svg" alt="RADAR-base Logo">
+    <img
+      class="logo"
+      src="assets/radar-logo.svg"
+      alt="RADAR-base Logo"
+    >
     <div class="label">
-      <div class="font-body-light">RADAR-base</div>
-      <div class="font-title">Studies</div>
+      <div class="font-body-light">RADAR-CNS Dashboard</div>
```
Comment (nivemaham) [inline src/app/studies/containers/studies-page.html]: Shouldn't it be RADAR-Base Dashboard

Comment (mpgxvii) [reply src/app/studies/containers/studies-page.html]: Okay, I'll change it.

File: src/app/studies/containers/studies-page.html [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
+      *ngIf="studiesIsLoaded$ | async"
+    >
+      <div class="grid-wrapper">
+        <div class="grid">
+          <div
+            class="study-card"
+            *ngFor="let study of studies$ | async"
+            (click)="openStudy(study.projectName)"
+          >
+            <h2 class="title">{{ study.projectName }}</h2>
```
Comment (nivemaham) [inline src/app/studies/containers/studies-page.html]: There was a request from Herc to send a[ prettified project name](https://github.com/RADAR-base/RADAR-RestApi/issues/98) to use for display purpose. So currently, the rest-api also contains `humanReadableProjectName`. You can show `study.humanReadableProjectName ? study.humanReadableProjectName : study.projectName`. (Note: This is change is yet to be released.)

File: src/app/studies/containers/studies-page.html [hunk 3]
```diff
@@ -1,35 +1,46 @@
...
+            <h4 class="subtitle">
+              <span>{{ study.id }}</span>
+              <span *ngIf="study.location">| {{ study.location }}</span>
```
Comment (nivemaham) [inline src/app/studies/containers/studies-page.html]: can you please add the space back before the `|`? `<span *ngIf="study.location"> | {{ study.location }}</span>`

**Review by nivemaham — APPROVED** [review]
LGTM

**Pull request merged**